subprojects/platform-native/src/testFixtures/groovy/org/gradle/nativeplatform/fixtures/binaryinfo/OtoolBinaryInfo.groovy

```diff
@@ -21,13 +21,15 @@ import org.gradle.nativeplatform.platform.internal.Architectures
 
 class OtoolBinaryInfo implements BinaryInfo {
     def binaryFile
+    private final List<String> environments
 
-    OtoolBinaryInfo(File binaryFile) {
+    OtoolBinaryInfo(File binaryFile, List<String> environments) {
```

Comment:
Is this actually needed for running dwarfdump now?  Is this related to DEVELOPER_DIR?

Reply:
I prefer using the tools from the Xcode we used to build. This is required to use the right tool version.